robin.user.js

```diff
@@ -17,9 +17,11 @@ function howLongLeft() { // mostly from /u/Yantrio
     var message = $(".robin-message--message", remainingMessageContainer).text();
     var time = new Date($(".robin--user-class--system:contains('approx') .robin-message--timestamp").attr("datetime"));
```

Comment:
That wouldn't belong in this PR. This is strictly a bugfix for the existing code, not a feature change.